lib/commons/dom/is-visible.js

```diff
@@ -156,11 +156,15 @@ function isVisible(el, screenReader, recursed) {
   }
 
   // hidden from visual users
+  const elHeight = parseInt(style.getPropertyValue('height'));
   if (
     !screenReader &&
     (isClipped(style) ||
       style.getPropertyValue('opacity') === '0' ||
-      (getScroll(el) && parseInt(style.getPropertyValue('height')) === 0))
+      (getScroll(el) && elHeight === 0) ||
+      (style.getPropertyValue('position') === 'absolute' &&
+        elHeight <= 1 &&
```

Comment:
I would suggest we make this `< 2` to allow for some rounding above 1.
```suggestion
        elHeight < 2 &&
```